lib/ui/compositing/scene_builder.cc

```diff
@@ -168,9 +170,17 @@ void SceneBuilder::pushBackdropFilter(
     Dart_Handle layer_handle,
     ImageFilter* filter,
     int blendMode,
+    Dart_Handle backdropId,
```

Comment:
Not sure what you mean?

Reply:
There is another argument to this function `blendMode` which is an `int`.  It didn't come from Dart as an int, it came in as a Dart_Handle and got converted somewhere higher in the stack trace.  It seems like we should be doing our calls to `Dart_IntegerToInt64` at that same layer instead of in 2 different places.

Reply:
Oh, I C. No that was converted via dart::ffi, because int conversion is easy. In this case, the int is nullable - so we have to pass through the entire Dart_Handle and null check it in c++. 

I don't want to use a sentinel like -1, because then I'd have to add runtime checks to the dart API that don't otherwise need to be there.